### DISCUSSION TITLE
Add Frame-by-Frame Scrubbing

### DISCUSSION THREAD

**Discussion by thealexmay** [SubtitleEdit/subtitleedit#8152] [Ideas]
It would be a game changer if we could get frame-by-frame scrubbing within SubtitleEdit. Ideally with the , key for 1 frame back and . key for 1 frame forward.

There are many times when I'm trying to sync a line to a specific cut and have to do it through trial and error where I would normally just scrub frame-by-frame to find the exact time of the cut and put that in.

Frame-by-frame scrubbing would save tons of time.

**Comment by darnn** [comment]
Go to Settings, Shortcuts, search for "frame".

**Reply by thealexmay** [reply]
You mean to tell me it's been here this whole time? Well, shit.

I've literally googled all over the place to figure it out and nothing ever came up. Thank you, friend.

**Reply by darnn** [reply]
You're welcome! I've always been of the more "poke around until you find something" persuasion, which worked out well with SE, but yeah, I guess it's not everyone's first instinct to read every single keyboard shortcut you can set.
